client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -78,16 +85,29 @@ class PodcastingDetails extends Component {
 			isRequestingCategories,
 			translate,
 		} = this.props;
+		const { isCoverImageUploading } = this.state;
+
+		const saveButtonDisabled =
+			isRequestingSettings || isSavingSettings || isRequestingCategories || isCoverImageUploading;
+		let saveButtonText;
+		if ( isCoverImageUploading ) {
+			saveButtonText = translate( 'Image uploading…' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Image uploading...' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).